Cubical/Experiments/Epi.agda

```diff
@@ -0,0 +1,31 @@
+{-# OPTIONS --cubical #-}
+module Cubical.Experiments.Epi where
+  open import Cubical.Data.Unit
+  open import Cubical.Foundations.Everything
+  open import Cubical.Functions.Surjection
+  open import Cubical.HITs.PropositionalTruncation
+
+  postulate
+    Ω-ext : ∀ {ℓ} {p q : hProp ℓ} → (p .fst → q .fst ) → (q .fst → p .fst) → p ≡ q
+
+  ⊤* : ∀ {ℓ} → hProp ℓ
+  ⊤* = Unit* , λ _ _ _ → tt*
```

Comment:
There is this: https://github.com/agda/cubical/blob/master/Cubical/Data/Unit/Properties.agda#L63

I'm not sure if the packaged up definition as an hProp is somewhere?

Reply:
Actually there is this: https://github.com/agda/cubical/blob/master/Cubical/Functions/Logic.agda#L107